ampersand-model.js

```diff
@@ -60,10 +88,18 @@ var Model = State.extend({
         wrapError(this, options);
 
         method = this.isNew() ? 'create' : (options.patch ? 'patch' : 'update');
-        if (method === 'patch') options.attrs = attrs;
+
+        if (method === 'patch') {
+            options.attrs = transformForPatch(new this.constructor(attrs).serialize(), attrs);
```

Comment:
It's left over from when I was originally modifying the model instead of its serialized form. That said, I'm getting some strange test failures if I switch it to `this.serialize()`. I'll look into that then add a commit to fix.